public/electron.js

```diff
@@ -0,0 +1,226 @@
+const {
+  app,
+  BrowserWindow,
+  shell,
+  ipcMain,
+  Menu,
+  // eslint-disable-next-line import/no-extraneous-dependencies
+} = require('electron');
+const path = require('path');
+const isDev = require('electron-is-dev');
+const openAboutWindow = require('about-window').default;
+const logger = require('./app/logger');
+const {
+  ICON_PATH,
+  PRODUCT_NAME,
+  escapeEscapeCharacter,
+} = require('./app/config/config');
+const isMac = require('./app/utils/isMac');
+
+let mainWindow;
+
+const createWindow = () => {
+  logger.debug('create window');
+
+  mainWindow = new BrowserWindow({
+    backgroundColor: '#F7F7F7',
+    minWidth: 880,
+    show: false,
+    movable: true,
+    webPreferences: {
+      nodeIntegration: false,
+      preload: `${__dirname}/app/preload.js`,
+      webSecurity: false,
+    },
+    height: 860,
+    width: 1280,
+  });
+
+  mainWindow.loadURL(
+    isDev || process.env.NODE_ENV === 'test'
+      ? 'http://localhost:3000'
+      : `file://${path.join(__dirname, './index.html')}`,
+  );
+
+  if (isDev) {
+    const {
+      default: installExtension,
+      REACT_DEVELOPER_TOOLS,
+      REDUX_DEVTOOLS,
+      // eslint-disable-next-line global-require
+    } = require('electron-devtools-installer');
+
+    installExtension(REACT_DEVELOPER_TOOLS)
+      .then((name) => {
+        logger.info(`added extension: ${name}`);
+      })
+      .catch((err) => {
+        logger.error(err);
+      });
+
+    installExtension(REDUX_DEVTOOLS)
+      .then((name) => {
+        logger.info(`added extension: ${name}`);
+      })
+      .catch((err) => {
+        logger.error(err);
+      });
+  }
+
+  mainWindow.once('ready-to-show', () => {
+    mainWindow.show();
+
+    ipcMain.on('open-external-window', (event, arg) => {
+      shell.openExternal(arg);
+    });
+  });
+};
+
+const macAppMenu = [
+  {
+    label: app.name,
+    submenu: [
+      { role: 'about' },
+      { type: 'separator' },
+      { role: 'services' },
+      { type: 'separator' },
+      { role: 'hide' },
+      { role: 'hideothers' },
+      { role: 'unhide' },
+      { type: 'separator' },
+      { role: 'quit' },
+    ],
+  },
+];
+const standardAppMenu = [];
+const macFileSubmenu = [{ role: 'close' }];
+const standardFileSubmenu = [
+  {
+    label: 'About',
+    click: () => {
+      openAboutWindow({
+        // asset for icon is in the public/assets folder
+        base_path: escapeEscapeCharacter(app.getAppPath()),
+        icon_path: path.join(__dirname, ICON_PATH),
+        copyright: 'Copyright © 2019 React',
```

Comment:
Can we try and make the name dynamic? Also, let's use © 2020 EPFL & Graasp Association. If we can do it in two lines:

```
Copyright © 2020 EPFL
Copyright © 2020 Graasp Association
```